gravitee-apim-console-webui/src/management/api/analytics/pathMappings/api-path-mappings-add-dialog/api-path-mappings-add-dialog.component.html

```diff
@@ -0,0 +1,76 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<h2 mat-dialog-title class="title">Add Path Mapping</h2>
+
+<mat-dialog-content class="content">
+  <mat-tab-group dynamicHeight mat-stretch-tabs (selectedTabChange)="onSelectedTab($event)">
+    <!-- Add Path Mapping -->
+    <mat-tab [label]="tabLabels.AddPathMapping">
+      <div class="content__url-tab">
+        <mat-dialog-content class="content" [formGroup]="pathFormGroup">
+          <mat-form-field class="content__path__form-field" appearance="fill">
+            <mat-label>Path mapping</mat-label>
+            <input
+              type="text"
+              aria-label="Path mapping input"
+              placeholder="/products/:productId"
+              matInput
+              formControlName="path"
+              required
+            />
+            <mat-error *ngIf="pathFormGroup.controls.path.hasError('required')">Path mapping is required.</mat-error>
+            <mat-error *ngIf="pathFormGroup.controls.path.hasError('isUnique')">Path mapping already exists.</mat-error>
+            <mat-error *ngIf="pathFormGroup.controls.path.hasError('pattern')"
+              >Wrong path mapping pattern. Example: /products/:productId</mat-error
+            >
+          </mat-form-field>
+        </mat-dialog-content>
+      </div>
+    </mat-tab>
+
+    <!-- Import Swagger -->
+    <mat-tab [label]="tabLabels.SwaggerDocument">
+      <div class="content__url-tab">
+        <mat-dialog-content class="content" [formGroup]="pathFormGroup">
+          <mat-radio-group formControlName="path" class="radio-group" data-testid="designer-template-parameter-case">
```

Comment:
Maybe we should align it in column and not in line no wdyt ?
```
.radio-group {
  display: flex;
  flex-direction: column;

  &__button {
    margin: 8px 0 0 0;
  }
}
```

Reply:
done